tools/cli.js

```diff
@@ -24,14 +24,64 @@ const log = (msg) => {
   if (userInput) rl.write(userInput);
 };
 
+function completeByFields(input, object) {
+  return Object.keys(object)
+    .filter(c => !c.startsWith('_') && (!input || c.startsWith(input)));
```

Comment:
What type can `input` have? If it's always a string, then the `!input` check is superfluous, `c.startsWith('')` will always return `true`.